### PR TITLE
Fix `onefuzz repro` bug - Remove managed identity from `CustomScriptExtension`

### PR DESCRIPTION
**Pull request by nharper285**
## Summary of the Pull Request

_What is this about?_
The `CustomScriptExtension` began failing due to the empty managed identity that's added to its configuration. Despite having this configuration for some time, the extension began failing abut three weeks ago (potentially due to an internal change). 

This PR removes the managed identity from the extension config. 

## PR Checklist
* [ ] Applies to work item: #2919
* [ ] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/onefuzz) and sign the CLI.
* [ ] Tests added/passed
* [ ] Requires documentation to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

## Info on Pull Request

_What does this include?_

## Validation Steps Performed

_How does someone test & validate?_

### CODE REVIEW

**Review by AdamL-Microsoft — APPROVED** [review]

**Review by stishkin — APPROVED** [review]

File: src/ApiService/ApiService/onefuzzlib/Extension.cs [hunk 1]
```diff
@@ -290,7 +290,7 @@ private sealed class Settings {
                 TypeHandlerVersion = "1.9",
                 AutoUpgradeMinorVersion = true,
                 Settings = new BinaryData(JsonSerializer.Serialize(new { commandToExecute = toExecuteCmd, fileUris = urlsUpdated }, _extensionSerializerOptions)),
-                ProtectedSettings = new BinaryData(managedIdentity)
+                ProtectedSettings = null
```
Comment (Porges) [inline src/ApiService/ApiService/onefuzzlib/Extension.cs:293]: You can probably remove this line entirely

**Review by Porges — APPROVED** [review]

**Review by chkeita — APPROVED** [review]

File: src/ApiService/ApiService/onefuzzlib/Extension.cs (hunk 1 above)
Comment (chkeita) [reply src/ApiService/ApiService/onefuzzlib/Extension.cs:293]: you can also remove the `managedIdentity` variable initialization as well

**Comment by mgreisen** [comment]
>                 ProtectedSettings = new BinaryData(managedIdentity)

don't we need to do this here as well?

---
Refers to: src/ApiService/ApiService/onefuzzlib/Extension.cs:318 in bb16de8. [](commit_id = bb16de8626c6f805d9cd1128bc6130d92c4a3e58, deletion_comment = False)

File: src/ApiService/ApiService/onefuzzlib/Extension.cs (hunk 1 above)
Comment (chkeita) [reply src/ApiService/ApiService/onefuzzlib/Extension.cs:293]: we should also remove this from the linux config too

Comment (nharper285) [reply src/ApiService/ApiService/onefuzzlib/Extension.cs]: Removed.

**Review by mgreisen — APPROVED** [review]

**Review by tevoinea — APPROVED** [review]

**Pull request merged**